pkg/client/proxy_replica.go

```diff
@@ -72,19 +74,21 @@ func (c *ProxyClient) ReplicaList(backendStoreDriver, engineName, volumeName,
 		return nil, errors.Wrap(err, "failed to list replicas for volume")
 	}
 
-	driver, ok := rpc.BackendStoreDriver_value[backendStoreDriver]
+	driver, ok := rpc.DataEngine_value[getDataEngine(dataEngine)]
```

Comment:
nit: Suggest creating a function for decoding `DecodeDataEngineEnum` (add to `imrpc` pkg), so there won't be a need to do conversion in each method.

```go
	dataEngine, ok := imrpc.DecodeDataEngineEnum(dataEngine)
	if !ok {
		return nil, fmt.Errorf("failed to list replicas for volume: invalid data engine %v", dataEngine)
	}

	defer func() {
		err = errors.Wrapf(err, "%v failed to list replicas for volume", c.getProxyErrorPrefix(serviceAddress))
	}()

	req := &rpc.ProxyEngineRequest{
		Address:            serviceAddress,
		EngineName:         engineName,
		Address:    serviceAddress,
		EngineName: engineName,
		// Deprecated
		BackendStoreDriver: rpc.BackendStoreDriver(driver),
		DataEngine:         dataEngine,
		VolumeName:         volumeName,
	}
```

Reply:
Currently, we need to decode `BackendStoreDriver` and `DataEngine`, so the codes will be length. I will add decode function after removing `BackendStoreDriver` in v1.7.0.